### PR TITLE
Add allow_unknown_urls

### PR DESCRIPTION
**Pull request by nritholtz**
Ability to suppress the default unknown URLs warning as per #744

### CODE REVIEW

File: lib/capybara/webkit/driver.rb [hunk 1]
```diff
@@ -33,6 +33,10 @@ def block_unknown_urls
       browser.block_unknown_urls
     end
 
+    def suppress_unknown_urls_warning
+      browser.suppress_unknown_urls_warning
```
Comment (jferris) [inline lib/capybara/webkit/driver.rb]: can this be implemented using just `driver.allow_url("*")`?

File: lib/capybara/webkit/driver.rb [hunk 2]
```diff
@@ -33,6 +33,10 @@ def block_unknown_urls
       browser.block_unknown_urls
     end
 
+    def suppress_unknown_urls_warning
```
Comment (jferris) [inline lib/capybara/webkit/driver.rb]: Thoughts on `allow_unknown_urls` instead?

**Comment by jferris** [comment]
Thanks! I had a couple comments about naming and implementation.

**Comment by nritholtz** [comment]
Hey @jferris, I've made what I think are the changes you wanted and also updated the specs/README accordingly. Let me know.

**Comment by jferris** [comment]
Thanks! I merged this as 6e485ca4.

**Pull request closed without merging**